qiita_pet/templates/study_base.html

```diff
@@ -143,6 +151,23 @@
     }
   }
 
+  function delete_study() {
```

Comment:
Did you modify this function in some way? or it just says that has been moved due to the merge conflict?

Please be careful solving merge conflicts, otherwise is really hard to review.